client/src/pages/Home/Footer/Footer.css

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
do we need this file for this part of the app?

Reply:
No, we don't need the following file.

Footer.js Footer.css I need to delete the folder.